site/en/docs/extensions/mv3/getstarted/index.md

```diff
@@ -272,9 +273,9 @@ Reload the extension to view the green button.
 
 ## Layer logic {: #logic }
 
-The extension now knows the popup should be available to users on [developer.chrome.com][23] and
-displays a colored button but needs logic for further user interaction. Update `popup.js` by adding
-the following to the end of the file.
+The extension now has a custom icon, a popup, and colors the popup button based on a value saved to
```

Comment:
It's strange grammar to say it has a thing, a thing, and verb phrase. Consider:
...has a custom icon and a popup. and it colors the...